build.gradle

```diff
@@ -165,6 +172,9 @@ configure(projectsWithFlags('java')) {
         // Reflections
         implementation libs.reflections
 
+        // Blockhound
+        implementation libs.blockhound
```

Comment:
Can we make it optional?

Reply:
I missed this 😅   Thanks for pointing this out 👍